lib/requester/request-wrapper.js

```diff
@@ -86,13 +91,55 @@ var _ = require('lodash'),
               options.proxy = proxyConfig.server.toString(),
               options.tunnel = proxyConfig.tunnel;
               _.set(options, 'request._._postman_proxy', proxyConfig);
+              _.set(options, 'request.proxy', proxyConfig);
             }
             return cb(null, options);
         }
         cb(null, options);
     },
 
     /**
+     * Resolves the content of a certificate from the file system,
+     * using the current file resolver.
+     *
+     * @param certificate The certificate to resolve
+     * @param resolver The file resolver to use
+     * @param cb Callback once resolve is done
+     */
+    resolveCertificateContents = function(certificate, resolver, cb) {
+        var keyPath = _.get(certificate, 'key.src'),
+            certPath = _.get(certificate, 'cert.src');
+
+        if (!_.isString(keyPath)) {
+            return cb(ERROR_MISSING_KEY_PATH);
```

Comment:
You're passing a string as the first argument to the callback, need to use `new Error` here I think